ref/test/src/interpolateTest.f90

```diff
@@ -55,7 +55,7 @@ program interpolateTest
           if (abs(interpArray(k, j) - 1.0) .gt. epsilon) then
             print *, "Failure"
             print *, k, j, interpArray(k, j), interpArray(k, j) - 1.0
-            stop -10
+            stop - 10
```

Comment:
I know the style guide says "spaces around operators", but I should probably clarify that it doesn't apply when signifying a negative number.  A subtle difference, that highlights how difficult it is to properly document a "style".

Would you mind changing this to a `-10` instead of a `- 10` as I think this looks a bit weird.

Reply:
Done.